automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/WorkspaceSummaryPage.scala

```diff
@@ -186,8 +186,9 @@ class CloneWorkspaceModal(implicit webDriver: WebDriver) extends FireCloudView {
       textField(workspaceNameInput).value = name
     }
 
-    def readPresetAuthDomain(): Option[String] = {
-      find(presetAuthDomain).map(_.text)
+    def readPresetAuthDomainGroup(groupName: String): Option[String] = {
+      await visible lockedAuthDomainGroup(groupName)
```

Comment:
I was going to suggest that this could be inlined:

`singleSel(await visible lockedAuthDomainGroup(groupName)).selection`

Unfortunately, unlike `await.enabled`, `await.visible` does not return the found element. 😞  However, that can be fixed 😃  by changing the implementation of `visible` to be like `enabled` but filtering on `isDisplayed`.

That said, I'm pretty much 50/50 on which way reads better so it's up to you whether or not you want to make this change.